mountpoint-s3/scripts/fio/read/seq_read_skip_read.fio

```diff
@@ -0,0 +1,14 @@
+[global]
+name=fs_bench
+bs=256m
+runtime=30s
+time_based
+group_reporting
+
+[seq_read_skip_read]
+size=2G
+rw=read:17m
```

Comment:
Could we put `17m_skip` in the name or something? I can imagine we might want other seek distances in the future.